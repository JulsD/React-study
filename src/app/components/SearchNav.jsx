import React from 'react';
import SearchSum from './SearchSum';
import SearchSort from './SearchSort';

class SearchNav extends React.Component {

  render() {
    const style = {
      width: '100%',
      display: 'flex'
    };
    return (
      <nav style={style}>
        <SearchSum />
        <SearchSort />
      </nav>
    );
  }

}

export default SearchNav;
