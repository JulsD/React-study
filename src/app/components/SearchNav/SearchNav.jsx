import React from 'react';
import SearchSum from './../SearchSum';
import SearchSort from './../SearchSort/SearchSort';

import styles from './search-nav.css';

class SearchNav extends React.Component {

  render() {
    return (
      <nav className = {styles.root}>
        <SearchSum />
        <SearchSort />
      </nav>
    );
  }

}

export default SearchNav;
