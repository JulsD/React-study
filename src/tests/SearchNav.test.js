import React from 'react';
import SearchNav from '../app/components/SearchNav';

test('SearchNav should render a <nav>', () => {
  const wrapper = shallow(
    <SearchNav />
  );
  expect(wrapper).toMatchSnapshot();
});
