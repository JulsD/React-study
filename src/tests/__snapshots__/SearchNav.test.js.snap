// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchNav should render a <nav> 1`] = `
<nav
  className="root"
/>
`;
